documentation/source/simulator_support.rst

```diff
@@ -242,12 +249,13 @@ Issues for this simulator
 -------------------------
 
 * `All issues with label category:simulators:questa <https://github.com/cocotb/cocotb/issues?q=is%3Aissue+-label%3Astatus%3Aduplicate+label%3Acategory%3Asimulators%3Aquesta>`_
-
+* Questa 2021.1 and later start to support the VHPI interface in addition to the proprietary FLI interface.
+  However, cocotb only supports the use of Questa through FLI.
```

Comment:
No longer true as of #2408. I should add a newsfragment, which I forgot in the PR. It's now more "they don't support us".

Reply:
I clarified this a bit. I really want to encourage users to continue using FLI until VHPI support is more stable and tested.